### ISSUE TITLE
Add plugin to message bus throw RuntimeException

### ISSUE THREAD

**Issue by oanqa** [prooph/service-bus-symfony-bundle#18]
According to the documentation, we can add plugin using service definition and register it under `prooph_service_bus.command_buses.acme_command_bus.plugins`.

But it throw `RuntimeException`[ProophServiceBusExtension](https://github.com/prooph/service-bus-symfony-bundle/blob/master/src/DependencyInjection/ProophServiceBusExtension.php#L130), because ContainerBuilder doesn't have definition when extension load.

Is this intentional, that we can only add plugin from the bundle itself?

**Comment by mablae** [comment]
Could you share the config snippet, where you configure the plugin service? 

The `prooph_service_bus.command_buses.acme_command_bus.plugins` expects an array of valid service 
references.

**Comment by oanqa** [comment]
i'm configuring inside `app/config/services.yml` like documentation.
```
services:
    acme.prooph.plugin.handle_command_strategy:
        class: Prooph\ServiceBus\Plugin\InvokeStrategy\HandleCommandStrategy
```

and inside my config,
```
prooph_service_bus:
    command_buses:
        acme_command_bus:
            plugins:
                - "acme.prooph.plugin.handle_command_strategy"
```

but if i'm using service defined in this bundle like `prooph_service_bus.handle_command_invoke_strategy`, it worked successfully.

I know i can use service tag plugin, and it worked, but in this situation i can't use my own plugin and register it via config.

**Comment by mablae** [comment]
I will check it and report back to you.

**Comment by oanqa** [comment]
thank you so much for your help,

I think this happens because, when symfony load extension it passing new container that doesn't have the defined service definition in other place. [MergeExtensionConfigurationPass](https://github.com/symfony/dependency-injection/blob/master/Compiler/MergeExtensionConfigurationPass.php#L48). CMIIW.
i'm using symfony 3.2.6.

**Comment by mablae** [comment]
I totally removed the check now. Can't even remember why I added it this way. 
It should work without the section now.

@kuloed Just released 0.4.1 Bugfix release. Thanks for your report and help. Could you check if it works for you now?
